_posts/en/newsletters/2021-03-10-newsletter.md

```diff
@@ -83,14 +82,14 @@ projects.
 
     - *Using time-based or height-based parameters:* the proposal
       describes the tradeoffs between setting its `start`, `timeout`,
-      and `minimum_activation` parameters using either times (based on
+      and `minimum_activation` parameters using either timestamps (based on
       the median of the previous 11 blocks) or block heights.  Using
-      times would result in the smallest and easiest to review patch to
+      timestamps would result in the smallest and easiest-to-review patch to
       Bitcoin Core.  Using heights would provide a bit more
       predictability, especially for miners, and would be compatible
       with other attempts using BIP8.
 
-    - *Myopic:* there was [concern][russell concern] that the proposal
+    - *Myopic:* Rusty Russell [expressed concern][russell concern] that the proposal
```

Comment:
We didn't attribute any of the other comments in this section, so I'd prefer not to single Rusty out.  (The main reason for not attributing is to encourage debating the ideas rather than the people proposing them.)

Reply:
Can we remove the quote marks in that case? It feels odd to quote something but not attribute it.

Reply:
Yes, feel free to remove the marks.